internal/gossip/gossip_redis.go

```diff
@@ -46,10 +47,15 @@ func (g *GossipRedis) Start() error {
 				err := g.Redis.ListenPubSubChannels(nil, func(channel string, b []byte) {
 					msg := newMessageFromBytes(b)
 					g.lock.RLock()
-					callbacks := g.subscribers[msg.key]
+					chans := g.subscriptions[msg.key]
 					g.lock.RUnlock()
-					for _, cb := range callbacks {
-						cb(msg.data)
+					// we never block on sending to a subscriber; if it's full, we drop the message
+					for _, ch := range chans {
+						select {
+						case ch <- msg.data:
+						default:
+							fmt.Println("dropped message", msg.key, string(msg.data))
```

Comment:
We have a logger, can we use it here?

Reply:
Heh, this was just debugging code I forgot to delete; but I'll make it a log.

Reply:
I think it is a useful log since it'll show when our value of 100 isn't sufficient